backend/models/test.model.ts

```diff
@@ -103,5 +126,9 @@ const TestSchema: Schema = new Schema(
   },
   { timestamps: true },
 );
+TestSchema.path("questions").validate(
```

Comment:
custom validator for mixed type - since mongoose doesn't have native support for 2D arrays